tests/test_runlib.py

```diff
@@ -32,7 +32,7 @@
 from in_toto.runlib import (in_toto_run, in_toto_record_start,
     in_toto_record_stop, record_artifacts_as_dict, _apply_exclude_patterns,
     _hash_artifact)
-from securesystemslib.interface import (generate_and_write_rsa_keypair,
+from securesystemslib.interface import (generate_and_write_unencrypted_rsa_keypair,
```

Comment:
nit: line length

Reply:
Fixed and squashed. Thanks for the review!

Reply:
Will merge once Travis/Appveyor has finished...